src/standalone-analytics.ts

```diff
@@ -2,23 +2,15 @@ import { Analytics, InitOptions } from './analytics'
 import { AnalyticsBrowser } from './browser'
 import { embeddedWriteKey } from './lib/embedded-write-key'
 
-type FunctionsOf<T> = {
-  [k in keyof T]: T[k] extends Function ? T[k] : never
-}
-
-type AnalyticsMethods = Pick<Analytics, keyof FunctionsOf<Analytics>>
-
-type StandaloneAnalytics = Analytics & {
+export type AnalyticsSnippet = Analytics & {
```

Comment:
Just curious, does this need to be exported? Fine if we do need to for our tests, just would have thought the global declaration would have us covered.